### PR TITLE
Don't translate Web Almanac and HTTP Archive

### PR DESCRIPTION
**Pull request by ksakae1216**
Part of #586

All replaced, Please review.

### CODE REVIEW

File: src/content/ja/2019/caching.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 part_number: IV
 chapter_number: 16
 title: キャッシング
-description: 2019Web年鑑のキャッシュの章は、キャッシュコントロール、有効期限、TTL、有効性、変化、Cookieの設定、アプリケーションキャッシュ、Service Worker、および機会について説明します。
+description: 2019Web Almanacのキャッシュの章は、キャッシュコントロール、有効期限、TTL、有効性、変化、Cookieの設定、アプリケーションキャッシュ、Service Worker、および機会について説明します。
```
Comment (bazzadp) [inline src/content/ja/2019/caching.md:5]: Should there be a space here? Or not required?
```suggestion
description: 2019 Web Almanacのキャッシュの章は、キャッシュコントロール、有効期限、TTL、有効性、変化、Cookieの設定、アプリケーションキャッシュ、Service Worker、および機会について説明します。
```

File: src/templates/ja/2019/chapters/caching.html [hunk 2]
```diff
@@ -10,7 +10,7 @@
...
 #}-->
 
-{% set metadata = {"part_number":"IV","chapter_number":16,"title":"キャッシング","description":"2019Web年鑑のキャッシュの章は、キャッシュコントロール、有効期限、TTL、有効性、変化、Cookieの設定、アプリケーションキャッシュ、Service Worker、および機会について説明します。","authors":["paulcalvano"],"reviewers":["obto","bkardell"],"translators":["ksakae"],"discuss":"1771","results":"https://docs.google.com/spreadsheets/d/1mnq03DqrRBwxfDV05uEFETK0_hPbYOynWxZkV3tFgNk/","queries":"16_Caching","published":"2019-11-11T00:00:00.000Z","last_updated":"2019-11-23T00:00:00.000Z","chapter":"caching"} %} {% block index %}
+{% set metadata = {"part_number":"IV","chapter_number":16,"title":"キャッシング","description":"2019Web Almanacのキャッシュの章は、キャッシュコントロール、有効期限、TTL、有効性、変化、Cookieの設定、アプリケーションキャッシュ、Service Worker、および機会について説明します。","authors":["paulcalvano"],"reviewers":["obto","bkardell"],"translators":["ksakae"],"discuss":"1771","results":"https://docs.google.com/spreadsheets/d/1mnq03DqrRBwxfDV05uEFETK0_hPbYOynWxZkV3tFgNk/","queries":"16_Caching","published":"2019-11-11T00:00:00.000Z","last_updated":"2019-11-23T00:00:00.000Z","chapter":"caching"} %} {% block index %}
```
Comment (bazzadp) [inline src/templates/ja/2019/chapters/caching.html:13]: Space needed or not?
```suggestion
{% set metadata = {"part_number":"IV","chapter_number":16,"title":"キャッシング","description":"2019 Web Almanacのキャッシュの章は、キャッシュコントロール、有効期限、TTL、有効性、変化、Cookieの設定、アプリケーションキャッシュ、Service Worker、および機会について説明します。","authors":["paulcalvano"],"reviewers":["obto","bkardell"],"translators":["ksakae"],"discuss":"1771","results":"https://docs.google.com/spreadsheets/d/1mnq03DqrRBwxfDV05uEFETK0_hPbYOynWxZkV3tFgNk/","queries":"16_Caching","published":"2019-11-11T00:00:00.000Z","last_updated":"2019-11-23T00:00:00.000Z","chapter":"caching"} %} {% block index %}
```

File: src/content/ja/2019/http2.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 part_number: IV
 chapter_number: 20
 title: HTTP/2
-description: HTTP/2、HTTP/2プッシュ、HTTP/2の問題、およびHTTP/3の採用と影響をカバーするWeb年鑑2019のHTTP/2章
+description: HTTP/2、HTTP/2プッシュ、HTTP/2の問題、およびHTTP/3の採用と影響をカバーするWeb Almanac2019のHTTP/2章
```
Comment (bazzadp) [inline src/content/ja/2019/http2.md:5]: Do we need a space here? Or should this be 2019 Web Almanac?
```suggestion
description: HTTP/2、HTTP/2プッシュ、HTTP/2の問題、およびHTTP/3の採用と影響をカバーするWeb Almanac 2019のHTTP/2章
```

File: src/templates/ja/2019/chapters/http2.html [hunk 4]
```diff
@@ -10,7 +10,7 @@
...
 #}-->
 
-{% set metadata = {"part_number":"IV","chapter_number":20,"title":"HTTP/2","description":"HTTP/2、HTTP/2プッシュ、HTTP/2の問題、およびHTTP/3の採用と影響をカバーするWeb年鑑2019のHTTP/2章","authors":["bazzadp"],"reviewers":["bagder","rmarx","dotjs"],"translators":["ksakae"],"discuss":"1775","results":"https://docs.google.com/spreadsheets/d/1z1gdS3YVpe8J9K3g2UdrtdSPhRywVQRBz5kgBeqCnbw/","queries":"20_HTTP_2","published":"2019-11-11T00:00:00.000Z","last_updated":"2019-11-23T00:00:00.000Z ","chapter":"http2"} %} {% block index %}
+{% set metadata = {"part_number":"IV","chapter_number":20,"title":"HTTP/2","description":"HTTP/2、HTTP/2プッシュ、HTTP/2の問題、およびHTTP/3の採用と影響をカバーするWeb Almanac2019のHTTP/2章","authors":["bazzadp"],"reviewers":["bagder","rmarx","dotjs"],"translators":["ksakae"],"discuss":"1775","results":"https://docs.google.com/spreadsheets/d/1z1gdS3YVpe8J9K3g2UdrtdSPhRywVQRBz5kgBeqCnbw/","queries":"20_HTTP_2","published":"2019-11-11T00:00:00.000Z","last_updated":"2019-11-23T00:00:00.000Z ","chapter":"http2"} %} {% block index %}
```
Comment (bazzadp) [inline src/templates/ja/2019/chapters/http2.html:13]:
```suggestion
{% set metadata = {"part_number":"IV","chapter_number":20,"title":"HTTP/2","description":"HTTP/2、HTTP/2プッシュ、HTTP/2の問題、およびHTTP/3の採用と影響をカバーするWeb Almanac 2019のHTTP/2章","authors":["bazzadp"],"reviewers":["bagder","rmarx","dotjs"],"translators":["ksakae"],"discuss":"1775","results":"https://docs.google.com/spreadsheets/d/1z1gdS3YVpe8J9K3g2UdrtdSPhRywVQRBz5kgBeqCnbw/","queries":"20_HTTP_2","published":"2019-11-11T00:00:00.000Z","last_updated":"2019-11-23T00:00:00.000Z ","chapter":"http2"} %} {% block index %}
```

File: src/content/ja/2019/caching.md (hunk 1 above)
Comment (ksakae1216) [reply src/content/ja/2019/caching.md:5]: Thanks, I fixed.

File: src/content/ja/2019/http2.md (hunk 3 above)
Comment (ksakae1216) [reply src/content/ja/2019/http2.md:5]: I fixed.

File: src/templates/ja/2019/chapters/caching.html (hunk 2 above)
Comment (ksakae1216) [reply src/templates/ja/2019/chapters/caching.html:13]: I fixd.

File: src/templates/ja/2019/chapters/http2.html (hunk 4 above)
Comment (ksakae1216) [reply src/templates/ja/2019/chapters/http2.html:13]: I fixed.

**Pull request merged**